framework/bin/run_mutation.pl

```diff
@@ -266,6 +272,17 @@ sub _run_mutation {
     $project->{prog_root} = "$root";
     $project->checkout_vid($vid) or die "Checkout failed";
 
+    my $num_excluded_mutants = 0;
+    if (defined $EXCL) {
+      system("cp $EXCL $root/") == 0 or die "Cannot copy exclude file";
```

Comment:
Should the exclude file be copied to $root/exclude.txt?

The file name `exclude.txt` is currently hard-coded in Project.pm and d4j-mutation. Should we change that?